src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {'My name is '}
+        {name}
+      </h2>
+      { age
+        && (
+        <p className="Person__age">
+          {'I am '}
+          {age}
+        </p>
+        )}
+      <p className="Person__partner">
+        { isMarried
+          ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
don't use nested ternaries, create a variable for `wife/husband`

Reply:
fixed all